scripts/tokenomics-emulation/ddc-smart-contract.ts

```diff
@@ -142,7 +142,24 @@ class DdcSmartContract {
         console.log(`The smart contract address is ${event.event.data[1]}\n`);
       }
     });
-  }
+   }
+  
+   /**
+   * Process the events to set code hash
+   * @param events Events
+   */
+    private async handleEventsForCodeHash(events) {
+      console.log("Handling Events");
+      events.forEach((event) => {
```

Comment:
Do not copy-paste handlers between SCs. Move to a separate class and reuse in both.

Reply:
Done. Created a new EventHandler class and handling there